src/test/java/de/hs_mannheim/informatik/ct/end_to_end/RoomControllerTest.java

```diff
@@ -48,42 +32,11 @@
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
 
-@SpringBootTest
-@AutoConfigureMockMvc
-@DirtiesContext(classMode = ClassMode.BEFORE_EACH_TEST_METHOD)
-@AutoConfigureTestDatabase(replace = Replace.ANY)
-public class RoomControllerTest {
-    @TestConfiguration
-    static class RoomControllerTestConfig {
-        @Bean
-        public RoomService service() {
-            return new RoomService();
-        }
-    }
-
-    @Autowired
-    private RoomService roomService;
-
-    @Autowired
-    private RoomVisitService roomVisitService;
-
-    @Autowired
-    private VisitorService visitorService;
-
-    @Autowired
-    private MockMvc mockMvc;
-
-    private final String TEST_ROOM_NAME = "123";
-    private String TEST_ROOM_PIN;
-    private final String TEST_ROOM_PIN_INVALID = "";
-    private final String TEST_USER_EMAIL = "1233920@stud.hs-mannheim.de";
-
-    @BeforeEach
-    public void setUp() {
-        Room room = new Room(TEST_ROOM_NAME, "A", 10);
-        TEST_ROOM_PIN = room.getRoomPin();
-        roomService.saveRoom(room);
-    }
+@SpringBootTest(properties = {
+        "allow_full_room_checkIn=true",
+        "warning_for_full_room=false"
+})
+public class RoomControllerTest extends BasicRoomControllerTest{
```

Comment:
the default room controller tests now have as fixed value for the environment variables, which are identical to the current settings from the config file.
room capacity is ignored and warnings are not displayed